xbmc/input/Action.cpp

```diff
@@ -11,13 +11,17 @@
 #include "ActionTranslator.h"
 #include "Key.h"
 
+CAction::CAction() :
+  m_id(ACTION_NONE)
+{
+  ClearAmount();
```

Comment:
redundant because of default member initialize `m_amount`

Reply:
thanks, I missed that one. updated again.